### DISCUSSION TITLE
Sprite 阴影截断了，按钮也被切了

### DISCUSSION THREAD

**Discussion by Evan-Huang-c** [HandyOrg/HandyControl#590] [Q&A]
显示如下
![image](https://user-images.githubusercontent.com/39617057/102473646-a99d6700-4092-11eb-9ab5-a3d27054da15.png)

使用样式如下：
`<hc:SimplePanel 
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:prism="http://prismlibrary.com/"
             prism:ViewModelLocator.AutoWireViewModel="True"
             xmlns:hc="https://handyorg.github.io/handycontrol">
    <Border Background="White" CornerRadius="20" Effect="{StaticResource EffectShadow5}">
        <Path Width="70" Height="70"  Fill="#f06632"/>
    </Border>
    <Button   Width="22" Height="22" Style="{StaticResource ButtonIcon}" Foreground="Black" hc:IconElement.Geometry="{StaticResource ErrorGeometry}" Padding="0" HorizontalAlignment="Right" VerticalAlignment="Top" Margin="0,-8,-8,0"/>
</hc:SimplePanel>`

文档上复制下来的

**Comment by Evan-Huang-c** [comment]
> 原来例子里给这个加了Padding

**Reply by GF-Huang** [reply]
你这 xaml 的排版，是给人看的吗？

**Reply by Evan-Huang-c** [reply]
是人就看，不是人就不看……
